### DISCUSSION TITLE
sqlalchemy.exc.ResourceClosedError: This result object does not return rows. It has been closed automatically.

### DISCUSSION THREAD

**Discussion by q1165600785** [sqlalchemy/sqlalchemy#9551] [Usage Questions]
hao i can get lastrowid or rowcount when i use bacth insert  and  bacth  update 

 result = conn.execute(insert(AntmodOneOne), insert_dict_list)
result=conn.execute(update(AntmodOneOne),insert_dict_list)

**Comment by q1165600785** [comment]
conn=sessionmaker(bind=self.enginer, expire_on_commit=False)()
above code i use seesion not enginer.connect()

**Comment by CaselIT** [comment]
Hi,

Can you provide a bit more context. Without it I can't help sorry
